corsikaio/subblocks/__init__.py

```diff
@@ -13,11 +13,10 @@
 from ..constants import RUNH_VERSION_POSITION, EVTH_VERSION_POSITION
 
 __all__ = [
-    'parse_event_header',
-    'parse_run_header'
-    'parse_cherenkov_photons',
-    'parse_particle_data',
-    'parse_longitudinal',
+    "parse_event_header",
+    "parse_run_header" "parse_cherenkov_photons",
```

Comment:
Missing comma error that was here before now turns up here.

Reply:
fixed